ui/app/utils/parse-pki-cert-oids.js

```diff
@@ -2,7 +2,7 @@
  * Copyright (c) HashiCorp, Inc.
  * SPDX-License-Identifier: MPL-2.0
  */
-import camelizeKeys from 'vault/utils/camelize-object-keys';
```

Comment:
I can't even find this under utils... ?? But nice replacement.

Reply:
It's a new one I added!